CMakeLists.txt

```diff
@@ -193,11 +184,10 @@ install(FILES
   DESTINATION src/Hydra)
   
 
-
 if(BUILD_DOCUMENTATION)
 
     if(NOT DOXYGEN_FOUND)
-        message(FALTAL_ERROR "Doxygen is needed to build the documentation.")
```

Comment:
FATAL_ERROR, not FALTAL_ERROR. I'm guessing you have always had Doxygen installed.

Reply:
Not at all. The documentation processing needs be explicitly requested passing   -DBUILD_DOCUMENTATION=ON 
to cmake command. I will correct the typo.

Reply:
I meant you always had Doxygen installed when you passed `-DBUILD_DOCUMENTATION=ON`